packages/common/src/doctype.ts

```diff
@@ -74,8 +74,6 @@ export interface DocMetadata {
     controllers: Array<string>
     schema?: string
     tags?: Array<string>
-
-    [index: string]: any // allow arbitrary properties (nonce, etc.)
```

Comment:
plz don't remove this. We want to support other things in the metadata (like family for example).

Reply:
👍